### PR TITLE
Adding level and experience

### PR DESCRIPTION
**Pull request by AevitasDragonkin**
Adds the pokemon level to:
- Gyms detail view
- Routes overview and detail view 
 
Adds the pokemon experience to:
- Gyms detail view
- Pokemon overview and detail view

### CODE REVIEW

**Comment by AevitasDragonkin** [comment]
Updated conflicts

File: pages/Gyms/main.html [hunk 1]
```diff
@@ -78,6 +80,8 @@ <h3>Pokémon Team</h3>
                         <a class="text-decoration-none" href="#!" data-bind="text: $data.name, attr: {href: `#!Pokemon/${$data.name}` }"></a>
                     </td>
                     <td class="align-middle" data-bind="text: $data.maxHealth.toLocaleString()"></td>
+                    <td class="align-middle" data-bind="text: $data.level.toLocaleString()"></td>
+                    <td class="align-middle" data-bind="text: (pokemonList.find(p => p.name == $data.name).exp * 1.5).toLocaleString()"></td>
```
Comment (CypherX) [inline pages/Gyms/main.html:84]:
```suggestion
                    <td class="align-middle" data-bind="text: (pokemonMap[$data.name].exp * 1.5).toLocaleString()"></td>
```
You can just use `pokemonMap` here and the other 2 places

Comment (AevitasDragonkin) [reply pages/Gyms/main.html]: Done

**Comment by CypherX** [comment]
Looks good, thanks

**Pull request merged**